INSTRUCTION.md

```diff
@@ -0,0 +1,12 @@
+## HOW shoude TO USE
+
+### -- 1. Oper mysql client  
+mysql -u your_username -p  
+
+### -- 2. Enter passwor: (from base owner)
```

Comment:
There is a typographical error in 'Enter passwor:'. It should be corrected to 'Enter password:'.